File: go/wam/6938.go
const WamPsRichOrderStatusMessageInconsistentPayloadReceived = 6938 // channel: private

type PsRichOrderStatusMessageInconsistentPayloadReceivedEvent struct {
	BusinessJid string `wam:"business_jid"` // field 1
	HasCurrencyChanged bool `wam:"has_currency_changed"` // field 2
	HasHeaderImageChanged bool `wam:"has_header_image_changed"` // field 3
	HasItemImageChanged bool `wam:"has_item_image_changed"` // field 4
	HasItemNameChanged bool `wam:"has_item_name_changed"` // field 5
	HasItemNumberChanged bool `wam:"has_item_number_changed"` // field 6
	HasItemPriceChanged bool `wam:"has_item_price_changed"` // field 7
	HasItemQuantityChanged bool `wam:"has_item_quantity_changed"` // field 8
	HasItemVariantChanged bool `wam:"has_item_variant_changed"` // field 9
}

File: rust/wam/6938.rs
pub const WAM_PS_RICH_ORDER_STATUS_MESSAGE_INCONSISTENT_PAYLOAD_RECEIVED: u32 = 6938;

#[derive(Debug, Default)]
pub struct PsRichOrderStatusMessageInconsistentPayloadReceivedEvent {
    /// field 1, wire `business_jid`
    pub business_jid: Option<String>,
    /// field 2, wire `has_currency_changed`
    pub has_currency_changed: Option<bool>,
    /// field 3, wire `has_header_image_changed`
    pub has_header_image_changed: Option<bool>,
    /// field 4, wire `has_item_image_changed`
    pub has_item_image_changed: Option<bool>,
    /// field 5, wire `has_item_name_changed`
    pub has_item_name_changed: Option<bool>,
    /// field 6, wire `has_item_number_changed`
    pub has_item_number_changed: Option<bool>,
    /// field 7, wire `has_item_price_changed`
    pub has_item_price_changed: Option<bool>,
    /// field 8, wire `has_item_quantity_changed`
    pub has_item_quantity_changed: Option<bool>,
    /// field 9, wire `has_item_variant_changed`
    pub has_item_variant_changed: Option<bool>,
}